CMakeLists.txt

```
@@ -174,6 +174,15 @@ ecbuild_add_option( FEATURE ECTRANS4PY
 
 ectrans_find_lapack()
 
+ecbuild_add_option( FEATURE FIELD_API
+                    DEFAULT OFF
+                    DESCRIPTION "Compile field API interface to ectrans"
+                   )
+
+if (HAVE_FIELD_API)
+    ecbuild_find_package(field_api)
+endif()
+
```

I would propose to simply add `field_api` as a required package of `FIELD_API`
```suggestion
ecbuild_add_option( FEATURE FIELD_API
                    DEFAULT OFF
                    DESCRIPTION "Compile field API interface to ectrans"
                    REQUIRED_PACKAGES "field_api"
                   )

```
(Needs testing, I'm not 100% certain of the syntax.)

---

done